Examples/Swift/FillPatternExample.swift

```diff
@@ -0,0 +1,86 @@
+import Mapbox
+
+@objc(FillPatternExample_Swift)
+
+class FillPattern_Swift: UIViewController, MGLMapViewDelegate {
+    var fillPatternImage: UIImage!
```

Comment:
Because this is only used in `didFinishLoadingStyle`, this doesn’t have to be a property of the view controller. Instead, we can do all of the setup in `didFinishLoadingStyle`.